data/json/proficiencies/chemistry.json

```diff
@@ -8,5 +28,65 @@
     "default_time_multiplier": 2,
     "default_fail_multiplier": 2.5,
     "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_mutagen",
+    "name": { "str": "Mutagen Production" },
+    "description": "Knowledge on the production of general mutagens.",
```

Comment:
I don't think this is a good proficiency. Mutagens should use the same science as everything else, this just unnecessarily sets them apart from other recipes. If they use the same science proficiencies as other stuff, it will be easier and more rewarding to gain science proficiencies, and it will tie mutagens more to the real world.